### DISCUSSION TITLE
[FR] Field layout for sections

### DISCUSSION THREAD

**Discussion by thupsi** [craftcms/cms#9783] [Ideas]
When doing content modelling, there are often common properties that should be shared between a section's entries (meaning they should be the same for all entries in a given section).  Or there are some properties that characterize the section as a whole.

When this need arises I usualy set-up an entry (or category), relate the new entry to the section (although as far as I know there is no straightforward way to do this through the CP without a plugin, [see discussion #9736](https://github.com/craftcms/cms/discussions/9736)) and assign the properties to this entry through custom fields.

While it's workable and by no means a showstopper, I can see two problems with this approach:

- The need to create an entry/category as a property container.
- It feels counterintuitive for editors to handle properties of the section itself through another entity.

Proposed solution:
Have a field layout for sections like the one for entries, categories, asset volumes etc.

I'm not sure of the difficulty in implementing this, or even if the Craft team would dim it worthwhile at this point, but I would love to hear other ways of handling this, or if there is a recommended "Craft" way I'm missing! Thanks!

**Comment by brandonkelly** [comment]
I have an idea I’m planning on investigating for Craft 5, which would solve this (along with your other idea #9736).

**Reply by thupsi** [reply]
Sounds intriguing! Can't wait to see what you come up with!
